nextflow_schema.json

```diff
@@ -38,27 +38,115 @@
                 }
             }
         },
-        "reference_genome_options": {
-            "title": "Reference genome options",
+        "database": {
+            "title": "Databases",
             "type": "object",
-            "fa_icon": "fas fa-dna",
-            "description": "Reference genome related files and options required for the workflow.",
+            "description": "Select databases to be run on all samples.",
+            "fa_icon": "fas fa-terminal",
             "properties": {
-                "genome": {
+                "pointfinder_database": {
                     "type": "string",
-                    "description": "Name of iGenomes reference.",
-                    "fa_icon": "fas fa-book",
-                    "help_text": "If using a reference genome configured in the pipeline using iGenomes, use this parameter to give the ID for the reference. This is then used to build the full paths for all required reference genome files e.g. `--genome GRCh38`. \n\nSee the [nf-core website docs](https://nf-co.re/usage/reference_genomes) for more details."
+                    "enum": [
+                        "Automatic Selection",
+                        "Enterococcus faecium",
+                        "Enterococcus faecalis",
+                        "Helicobacter pylori",
+                        "Salmonella",
+                        "Campylobacter",
+                        "Escherichia coli"
+                    ],
+                    "description": "Select a single Pointfinder database to use on all samples (overriding metadata option). Validated Organisms: Enterococcus faecium, Enterococcus faecalis, Helicobacter pylori, Salmonella, Campylobacter, Escherichia coli"
                 },
-                "igenomes_ignore": {
-                    "type": "boolean",
-                    "description": "Do not load the iGenomes reference config.",
-                    "fa_icon": "fas fa-ban",
-                    "hidden": true,
-                    "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
+                "plasmidfinder_database": {
+                    "type": "string",
+                    "enum": ["All", "gram_positive", "enterobacteriales"],
+                    "description": "The database type to use for plasmidfinder {gram_positive, enterobacteriales}. Defaults to using all available database types to search for plasmids. [All]."
+                },
+                "mlst_scheme": {
+                    "type": "string",
```

Comment:
Could you switch this to an `enum` type and include the list of all schemes (that is, directory names from https://github.com/tseemann/mlst/tree/master/db/pubmlst)?

Reply:
Sounds good! I added it

Reply:
Thanks so much.